igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import _thread
```

Comment:
Are you sure you want the low level approach? There are high level libraries for that. Especially with Python3 there is `concurrent` module.

Reply:
functions and modules starting with underscore are not meant to be used outside. They may disappear, change its behaviour at any time.

Reply:
changed to multiprosessing and simpyfied